content/en/blog/_posts/2024-09-20-kubeadm-v1beta4.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: blog
+title: 'Kubernetes v.31: kubeadm v1beta4'
+date: 2024-09-20
```

Comment:
is 20th of September the release date for 1.31?
https://github.com/kubernetes/sig-release/tree/master/releases/release-1.31
says 13th of August as release date.

Reply:
The date may need to be confirmed by release team.  I will send to the slack channel later.

Reply:
@pacoxu how about August 23rd? The release is set to be out aug 13th and we have several blogs in our [schedule](https://github.com/orgs/kubernetes/projects/183/views/6) already.

Reply:
> @pacoxu how about August 23rd? The release is set to be out aug 13th and we have several blogs in our [schedule](https://github.com/orgs/kubernetes/projects/183/views/6) already.

Thanks. Updated.